tests/test_main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from app.main import is_isogram
+
+
+def test_empty_string():
+    assert is_isogram("")
+
+
+def test_is_isogram_should_be_equal_true():
+    values = ["unCopyrIghtAbLe", "aMbiDextrOuslY", "plaYGrounDs"]
+    actual = []
+    for word in values:
+        actual.append(is_isogram(word))
+
+    assert all(actual)
```

Comment:
Bad test name, it would be better to name `test_no_repeated_letters`. Use mark parametrize here.

Reply:
Fixed